src/VaultSharp/V1/AuthMethods/Kerberos/IKerberosAuthMethod.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace VaultSharp.V1.AuthMethods.Kerberos
+{
+    /// <summary>
+    /// Kerberos Auth method.
+    /// </summary>
+    public interface IKerberosAuthMethod
```

Comment:
@rajanadar : What is the purpose of these interfaces?

Reply:
@roederja2  It is to provide management apis around those Auth backends in the future. The <Info> suffixed interfaces are purely for credential supply.